funnel/views/update.py

```diff
@@ -99,8 +110,10 @@ def loader(self, profile, project, update):
     @route('', methods=['GET'])
     @render_with('update_details.html.jinja2')
     def view(self):
+        if not self.obj.current_roles.reader and self.obj.state.WITHDRAWN:
```

Comment:
`reader` can be in `required_roles`?

Reply:
Drafts are publicly visible if you have the URL. Withdrawn updates are not. This distinction is necessary because we send out email notifications for published updates, so withdrawing is meaningless. Everyone who got the link can still open it.

If we change it so even drafts are not visible, this can go into the decorator.